web/wp-content/themes/dctx-v2/assets/bower_components/jquery/src/traversing/var/dir.js

```diff
@@ -0,0 +1,22 @@
+define( [
+	"../../core"
+], function( jQuery ) {
+
+"use strict";
+
+return function( elem, dir, until ) {
+	var matched = [],
+		truncate = until !== undefined;
+
+	while ( ( elem = elem[ dir ] ) && elem.nodeType !== 9 ) {
+		if ( elem.nodeType === 1 ) {
+			if ( truncate && jQuery( elem ).is( until ) ) {
+				break;
+			}
+			matched.push( elem );
+		}
+	}
+	return matched;
+};
```

Comment:
:no_entry_sign: **Error**: Multi-line function call not indented correctly; expected 4 spaces but found 0 (*PEAR.Functions.FunctionCallSignature.Indent*).